scripts/pr_review_checklist.py

```
@@ -0,0 +1,172 @@
+        default="",
+        help="Path to JSON file containing array of changed file paths",
+    )
+    args = parser.parse_args()
+
+    files = list(args.files)
+    if args.json_files:
+        try:
+            parsed = json.loads(args.json_files)
+            if isinstance(parsed, list):
+                files.extend([str(f) for f in parsed])
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)